src/Microsoft.Fast.Components.FluentUI/TextFieldType.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Fast.Components.FluentUI
+{
+    public enum TextFieldType
+    {
+        Text,
+        Email,
+        Password,
+        Tel,
```

Comment:
Do you think it would be more intuitive and consistent to spell out 'Tel' to 'Telephone'?

Reply:
The most used is Phone rather than Tel

Reply:
The way it works now with all the `enum`s in the library is there is a helper function that  'translates' the value to lowercase and inserts that string in the component variable. For an input to be 'phone enhanced' the value of the type needs to be 'tel'. That is why I used 'Tel' in the `enum`